src/Xlsx/RowIterator.php

```diff
@@ -124,6 +124,9 @@ public function next()
                             $this->valueTransformer->transform($this->xml->readString(), $type, $style)
                         );
                         break;
+                    case 'is':
+                        $rowBuilder->addValue( $columnIndex, $this->xml->readString() );
+                        break;
```

Comment:
Current spreadsheet parser is unable to process excel inline strings. It just ignores them and absolutely legitimate excel xlsx document looks empty.

Example of inline string cell:

<c r="A1">
<is>
<t>Hello, I am an Inline String !!!</t>
</is>
</c>